### PR TITLE
Fixed empty `ScriptLobbyDropdown` description crashing the game despite being optional

### LINKED ISSUE #21025: OpenRA crashes when `ScriptLobbyDropdown` doesn't define a description
Translation code crashes, description is null by default and it is not picked up by the linter. Should description be optional?

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/OpenRA/issues/21025

### CODE REVIEW

File: OpenRA.Game/Traits/TraitsInterfaces.cs [hunk 1]
```diff
@@ -550,7 +550,7 @@ public class LobbyOption
 		{
 			Id = id;
 			Name = map.GetLocalisedString(name);
-			Description = map.GetLocalisedString(description);
+			Description = description != null ? map.GetLocalisedString(description) : null;
```
Comment (anvilvapre) [inline OpenRA.Game/Traits/TraitsInterfaces.cs:553]: use `??`?

**Review by anvilvapre — APPROVED** [review]

File: OpenRA.Game/Traits/TraitsInterfaces.cs (hunk 1 above)
Comment (PunkPun) [reply OpenRA.Game/Traits/TraitsInterfaces.cs:553]: that won't work

**Review by PunkPun — APPROVED** [review]

**Comment by PunkPun** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/1fa76d70376196fb1eb79ea1d51b67672cb8f64a...6bcd88fc890eb60efefd29a9f9e02b8cdefb0db0)

https://github.com/OpenRA/OpenRA/compare/8b3e7bec2a9c...f4c186b7a6b5

File: OpenRA.Game/Traits/TraitsInterfaces.cs (hunk 1 above)
Comment (anvilvapre) [reply OpenRA.Game/Traits/TraitsInterfaces.cs:553]: mixing languages `https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining`

**Pull request merged**